common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -355,7 +359,35 @@ public Bitmap getDefaultVideoPoster() {
                 // We will return a 10x10 empty image, instead of the default grey playback image. #2424
                 return Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888);
             }
+
+            /**
+             * Capture console messages and log them using our logger.
+             *
+             * @param consoleMessage The console message from the WebView
+             * @return returns true if we handled the message. False will let the default handler handle it.
+             */
+            @Override
+            public boolean onConsoleMessage(@NonNull final ConsoleMessage consoleMessage) {
+                // Note: Decide what we are interested in logging and what level.
+                super.onConsoleMessage(consoleMessage);
+                final String methodTag = TAG + ":onConsoleMessage";
+                final String errorMessage = consoleMessage.message() + " -- From line " +
+                        consoleMessage.lineNumber() + " of " + consoleMessage.sourceId();
+                switch (consoleMessage.messageLevel()) {
+                    case ERROR:
+                        Logger.errorPII(methodTag, errorMessage, null);
+                        break;
+                    case WARNING:
+                        Logger.warnPII(methodTag, errorMessage);
+                        break;
+                    default:
+                        Logger.infoPII(methodTag, errorMessage);
+                }
+                return true;
```

Comment:
I'm a little concerned about how many additional logs this will create in our logs. Have we tested this to see how much noise this adds?

Reply:
removed